Modules/socketmodule.c

```diff
@@ -2869,11 +2883,13 @@ sock_accept(PySocketSockObject *s, PyObject *Py_UNUSED(ignored))
     newfd = ctx.result;
 
 #ifdef MS_WINDOWS
+#ifdef MS_WINDOWS_DESKTOP
```

Comment:
In 10.0.19041, `SetHandleInformation()` is available for `APP`, `SYSTEM`, and `GAMES`. Your experiment with modifying `HANDLE_FLAG_INHERIT` on Xbox failed with `ERROR_ACCESS_DENIED`, right? In that case, I suggest supporting everything but `GAMES`.

```suggestion
#if defined(MS_WINDOWS_APP) || defined(MS_WINDOWS_DESKTOP) || defined(MS_WINDOWS_SYSTEM)
#ifndef HANDLE_FLAG_INHERIT
#define HANDLE_FLAG_INHERIT 0x00000001
#endif
```

Reply:
Yes it did fails with `ERROR_ACCESS_DENIED`